src/scripts.rs

```diff
@@ -47,6 +47,11 @@ use serde::de;
 /// <https://github.com/bitcoin/bitcoin/blob/4a540683ec40393d6369da1a9e02e45614db936d/src/primitives/transaction.h#L87-L89>
 pub const SEQUENCE_LOCKTIME_MASK: u32 = 0x00_00_ff_ff;
 
+// Limiting the number of managers and stakeholders to 20 allows us to use a multi() descriptor for both
+// the deposit and CPFP descriptor, greatly simplifying the implementation of revaultd until bitcoind supports Miniscript.
```

Comment:
nit: for posterity, can you also add as rationale that having bounds also makes it easier to reasonably test most deployment parameters?